### PR TITLE
Release Flink Table Store 0.2.1

### PR DESCRIPTION
**Pull request by JingsongLi**

### CODE REVIEW

File: _posts/2022-09-30-release-table-store-0.2.1.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+- JingsongLi:
+  name: "Jingsong Lee"
+
+excerpt: The Apache Flink Community is pleased to announce a bug fix release for Flink Table Store 0.2.
+
+---
+
+The Apache Flink Community is pleased to announce the first bug fix release of the Flink Table Store 0.2 series.
+
+This release includes 14 bug fixes, vulnerability fixes, and minor improvements for Flink Table Store 0.2.
```
Comment (carp84) [inline _posts/2022-09-30-release-table-store-0.2.1.md]: Please check and confirm whether [FLINK-29554](https://issues.apache.org/jira/browse/FLINK-29554) and [FLINK-29345](https://issues.apache.org/jira/browse/FLINK-29345) should be included and update the number accordingly if not.

File: _posts/2022-09-30-release-table-store-0.2.1.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29241'>FLINK-29241</a>] -         Can not overwrite from empty input
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29273'>FLINK-29273</a>] -         Page not enough Exception in SortBufferMemTable
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29278'>FLINK-29278</a>] -         BINARY type is not supported in table store
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29295'>FLINK-29295</a>] -         Clear RecordWriter slower to avoid causing frequent compaction conflicts
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29345'>FLINK-29345</a>] -         Too many open files in table store orc writer
```
Comment (carp84) [inline _posts/2022-09-30-release-table-store-0.2.1.md]: It seems the fix version of [FLINK-29345](https://issues.apache.org/jira/browse/FLINK-29345) is 0.3.0 only, please check and confirm whether this is included in 0.2.1

**Review by carp84 — APPROVED** [review]
Overall LGTM. Please check my inline comments.

**Review by wuchong — COMMENTED** [review]
+1 after @carp84 's comments are addressed.

**Pull request merged**